po/sr.po

```diff
@@ -268,10 +268,10 @@ msgstr "прошли месец"
 #. Translators: This message will never be used for less than
 #. 2 months ago nor for more than one year ago. See the comment above.
 #: ../src/gnome_abrt/tools.py:75
-#, python-brace-format
+#, python-brace-format, fuzzy
 msgid "{0:d} month ago"
 msgid_plural "{0:d} months ago"
-msgstr[0] "пре једног месеца"
+msgstr[0] "пре {0:d} месеца"
 msgstr[1] "пре {0:d} месеца"
 msgstr[2] "пре {0:d} месеци"
 
```

Comment:
These changes should be done in Zanata and then pulled. Otherwise translators will not know about required changes and will overwrite them with the old translation. If this is urgent and Serbian translators are unresponsive then you can fix the translation on your own. I think I have permission to do this. You should not modify ```*.po``` files directly unless this is really exxxtremely urgent.

Reply:
Indeed you’re right. I got permissions yesterday, so I’ll try to just fuzzy it in Zanata. It’s fairly urgent in that msgfmt freaks out and makes the build fail.

Reply:
Okay, I did that and pulled translations into master. Gonna rebase now.

Reply:
> Indeed you’re right. I got permissions yesterday, so I’ll try to just fuzzy it in Zanata.

Why fuzzy? The translations marked as fuzzy will be completely ignored. Replacing ```"једне"``` with ```"{0:d}"``` where the number is 1 is most likely correct. I would doubt the change the other way (unless you know the language) but not this. Finally, this message will never be used with the number 1, ```"Last month"``` will be used instead. It is there only because it is required by the gettext API and only because there are few languages where this is really required.

Reply:
My bad, I didn’t know that fuzzy translations are skipped. Looking at Nautilus translations, for instance, it seems that the case for the plural form is likely correct, but I don’t know the language, hence the fuzzy flag.

Reply:
And, yes, I know it won’t be used.